pkg/gofr/migration/redis.go

```diff
@@ -156,5 +156,7 @@ func (d redisMigrator) commitMigration(c *container.Container, data migrationDat
 func (d redisMigrator) rollback(c *container.Container, data migrationData) {
 	data.RedisTx.Discard()
 
+	migrationVersion := strconv.FormatInt(data.MigrationNumber, 10)
```

Comment:
@kailashchoudhary11 - it isn't necessarily required to convert type of migration number, we can directly log the value as well.